src/controllers/todo-controller.ts

```diff
@@ -0,0 +1,56 @@
+import { BaseController } from './base-controller';
+import { NextFunction, Response, Router } from 'express';
+import { Validation } from '@helpers';
+import { TodoItem } from '@models';
+import {
+  AppContext,
+  Errors,
+  ExtendedRequest,
+  ValidationFailure,
+} from '@typings';
+import {
+  createTodoValidator
+} from '@validators';
+
+export class TodoController extends BaseController {
```

Comment:
Class name should be `TodoItemController`.

Reply:
updated